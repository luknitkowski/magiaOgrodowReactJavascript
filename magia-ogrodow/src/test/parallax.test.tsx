import renderer from 'react-test-renderer'

import imageAboutUsParallax from '../../src/images/dzialka.jpg';
import Parallax from '../components/parallax';

const propsParallax = {
    text: "", 
    image: imageAboutUsParallax
}

test('Parallax renders correctly', () => {
  const parallax = renderer.create(<Parallax {...propsParallax} />).toJSON();
  expect(parallax).toMatchSnapshot();
});

