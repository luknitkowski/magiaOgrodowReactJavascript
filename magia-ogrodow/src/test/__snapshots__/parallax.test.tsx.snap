// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Parallax renders correctly 1`] = `
<div
  className="sc-bdfBwQ cnTKxZ"
>
  <h1
    className="sc-gsTCUz iyjhuT"
  >
    
  </h1>
</div>
`;
